docs/using-radicle/overview.md

```diff
@@ -5,28 +5,28 @@ title: Overview
 
 ## How do I collaborate on Radicle?
 
-As a result of its [design][dr], Radicle Link supports a bazaar-style development model
-where there is no single canonical view of any project, but multiple upstreams
+Radicle Link is designed for bazaar-style development. This means that there is
+no single canonical view (e.g. master) of any project, but multiple upstreams
 maintained by maintainers and contributors that exchange patches with each
 other.
 
 Within the same project, two people will have subjective (and often diverging
 views), but *your* view of the project becomes the sum of all other views of the
-project. This means that when you find and follow a project on Radicle, you are
-not only able to replicate the maintainer's view of the project, but all other
+project. This means that when you find and follow a project on Radicle, you
+don't just replicate the maintainer's view of the project, but all other
```

Comment:
I don't this is true. AFAIK there is no implicit replication, every peers view has to be explicitly tracked. We should confirm.

Reply:
I thought there was... I'll remove for now to unblock this PR.